File: basic/src/main/java/com/yuanshenbin/basic/util/SPProxy.java
package com.yuanshenbin.basic.util;

import com.tencent.mmkv.MMKV;

/**
 * author : yuanshenbin
 * time   : 2022-10-21
 * desc   :
 */
public interface SPProxy {
    MMKV getSP(String key);
}

File: basic-kotlin/src/main/java/com/yuanshenbin/basic/util/SPProxy.kt
package com.yuanshenbin.basic.util

import com.tencent.mmkv.MMKV

/**
 * author : yuanshenbin
 * time   : 2022-10-21
 * desc   : 代理sp，这样可以创建多个不一样的实力
 */
interface SPProxy {
    fun filter(key: String?): MMKV?
}